app/views/submissions/show.html.erb

```diff
@@ -0,0 +1,11 @@
+<div>
+  <h2><%=@submission.title%></h2>
+  <h3><%=@submission.submitted_to%></h3>
+  <h3>Poems in this submission</h3>
+  <ul>
+    <% if @submission&.poems&.each do |poem| %>
+      <li><%=poem.title%></li>
+      <% end %>
```

Comment:
Ah, I see it now. I think there was another comment somewhere similar to this, but we're in the show action so we can assume `@submission` exists here, and because `poems` is a `has_many` association, it shouldn't be `nil` either. I think this could simplified to

```
<% @submission.poems.each do |poem| %>
  <li><%=poem.title%></li>
<% end %>
```